1.8/usage/service-discovery/haproxy-adminrouter.md

```diff
@@ -0,0 +1,107 @@
+---
+post_title: Configuring HAProxy in Front of Admin Router
+nav_title: HAProxy in Front of Admin Router
+menu_order: 400
+---
+
+You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/docs/1.8/overview/concepts/#adminrouter).
```

Comment:
> What is the reason for doing this? Might be helpful to discuss that.

There is a plethora of reasons. Some motivation is performance-related, some reasons may be policy-guided, other reasons may not even be good reasons, and other reasons are security-related in the widest sense. Overall, there are so many reasons (most of them unrelated to certificates), that I wouldn't go into mentioning specific reasons here on that page.

> If I am correct and it is for the self-signed certificate issue, then I can add a link to this page from my other section.

This help page was actually written with a motivation quite different from certificates. But yes, the solution we describe here can also be used to let someone take full control over the certificates presented to web browsers when accessing DC/OS via HTTPS through the proxy.

So, having a link from somewhere else discussing this certificate-related issue _to_ this help page here sounds perfectly fine.